tests/testcontainers-env/src/main/java/org/alfresco/tas/AlfrescoStackInitializer.java

```diff
@@ -102,7 +103,7 @@ public void initialize(ConfigurableApplicationContext configurableApplicationCon
         startOrFail(transformCore, transformRouter);
 
         // We don't want Kibana to run on our CI, but it can be useful when investigating issues locally.
-        if (Objects.equals(System.getProperty("kibana"), "true"))
+        if (getSystemProperty("kibana", "false").equals("true"))
```

Comment:
Flipping equals here would be safer in case of NPE.
```suggestion
        if ("true".equals(getSystemProperty("kibana", "false")))
```

Reply:
`getSystemProperty` returns the default value if the property isn't specified rather than `null` - i.e. it will be `false`.